<ui version="4.0" >
 <class>CommitteeInformationObject</class>
 <widget class="QWidget" name="CommitteeInformationObject" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Committee Information</string>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>11</y>
     <width>627</width>
     <height>368</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7" >
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="committeeNameLabel" >
        <property name="text" >
         <string>&lt;&lt;COMMITTEE NAME>></string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" >
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLabel" name="residentTableLabel" >
              <property name="text" >
               <string>Residents on this committee:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="residentSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>118</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="constituentView" />
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QPushButton" name="promoteSecretaryButton" >
              <property name="text" >
               <string>Promote to secretary</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="promoteChairButton" >
              <property name="text" >
               <string>Promote to chair</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="printCommitteeButton" >
          <property name="text" >
           <string>Print committee list</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="dividingLine" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" >
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <item>
             <widget class="QLabel" name="taskTableLabel" >
              <property name="text" >
               <string>Tasks:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="taskSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="taskView" />
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QPushButton" name="markTaskCompletedButton" >
              <property name="text" >
               <string>Mark as completed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addTaskButton" >
              <property name="text" >
               <string>Add Task</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteTaskButton" >
              <property name="text" >
               <string>Delete Task</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="printTaskButton" >
          <property name="text" >
           <string>Print task list</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" >
      <item>
       <spacer name="closeButtonSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton" >
        <property name="text" >
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
